elementary/monitor/api/tests/schema.py

```diff
@@ -50,6 +53,12 @@ class TestMetadataSchema(BaseModel):
     status: str
     test_created_at: Optional[str] = None
     days_diff: float
+    invocations_order: int
```

Comment:
I don't understand what does this represent.

Reply:
So this is part of the change in unifying the test queries.
Until now we had 3 different queries that were the same except the final filters (where clause).
We ordered the result of each test from the newest to oldest (invocations order), and we used that on the last filtering to get the latest tests runs + invocations for each test.

Now that we filter the results outside of the query (in the python), we need to return the invocations order from the query 🙂

Reply:
Does the `invocations_order` represent the `row_number` in the ordered result-set (`order by generated_at`)?
If so, why does the object itself need to be aware of its position in a list?
I think it makes more sense to pass around the ordered list when the order matters rather than store it on the object.

Reply:
Because we use `rank` method and not `row_number` method at the sql to order the invocations (we can have 2 different test results with 1 as the invocation order) we need to be aware of it 🙂

Reply:
Thanks for explaining.
Can we rename this to `invocations_rank_index`?
I think we should clarify that this represents an index within a set.

Reply:
Yeah sure!